pycheribuild/projects/cross/apache.py

```diff
@@ -129,3 +129,34 @@ def configure(self, **kwargs):
                          cwd=self.build_dir / "server")
             self.run_cmd(str(self.host_CC), "gen_test_char.lo", "-o",
                          "gen_test_char", cwd=self.build_dir / "server")
+
+class BuildSSLProc(CrossCompileCMakeProject):
+    target = "sslproc"
+
+    repository = GitRepository("https://github.com/CTSRD-CHERI/sslproc.git")
+
+    native_install_dir = DefaultInstallDir.IN_BUILD_DIRECTORY
+
+    @classmethod
+    def setup_config_options(cls, **kwargs):
+        super().setup_config_options(**kwargs)
+        cls.build_tests = cls.add_bool_option("build-tests", help="Build tests",
+                                              show_help=True)
+
+    def setup(self):
+        super().setup()
+        if self.build_tests:
+            self.add_cmake_options(BUILD_TESTS=True)
+
+class BuildSSLProcApache(BuildApache):
+    target = "apache-sslproc"
+    default_directory_basename = "apache"
```

Comment:
Should this not be `repository = ReuseOtherProjectRepository(BuildApache, do_update=True)` instead?

Reply:
Whichever is preferred?  (I don't care either way)

Reply:
Using this does let me remove `build_suffix` so that does seem cleaner.